test/e2e/infra/secondary_network/virtual-network-instance-crd.yml

```diff
@@ -0,0 +1,63 @@
+---
+apiVersion: "k8s.cni.cncf.io/v1"
+kind: NetworkAttachmentDefinition
+metadata:
+  name: virtual-net1
+  namespace: kube-system
```

Comment:
based on an earlier discussion, I believe `namespace: kube-system` should be removed from all `NetworkAttachmentDefinition` CRs in this file

Reply:
yes. taken care at the yaml as well now.